pkg/ossm/smcp_install.go

```diff
@@ -94,17 +94,17 @@ func TestSMCPInstall(t *testing.T) {
 		util.Shell(`oc get -n %s pods`, meshNamespace)
 	})
 
-	t.Run("smcp_test_uninstall_2.1", func(t *testing.T) {
+	t.Run("smcp_test_uninstall_2.2", func(t *testing.T) {
 		defer util.RecoverPanic(t)
-		util.Log.Info("Delete SMCP v2.1 in ", meshNamespace)
+		util.Log.Info("Delete SMCP v2.2 in ", meshNamespace)
 		util.KubeDeleteContents(meshNamespace, smmr)
-		util.KubeDeleteContents(meshNamespace, util.RunTemplate(smcpV21_template, smcp))
+		util.KubeDeleteContents(meshNamespace, util.RunTemplate(smcpV22_template, smcp))
 		time.Sleep(time.Duration(40) * time.Second)
 	})
 
-	t.Run("smcp_test_install_2.0", func(t *testing.T) {
+	t.Run("smcp_test_install_2.1", func(t *testing.T) {
 		defer util.RecoverPanic(t)
-		util.Log.Info("Create SMCP v2.0 in namespace ", meshNamespace)
+		util.Log.Info("Create SMCP v2.1 in namespace ", meshNamespace)
 		util.ShellMuteOutputError(`oc new-project %s`, meshNamespace)
 		util.KubeApplyContents(meshNamespace, util.RunTemplate(smcpV20_template, smcp))
```

Comment:
This install 2.1 step needs to run smcpV21_template

Reply:
done!